### DISCUSSION TITLE
How to access the global configuration or repository details from within a manager

### DISCUSSION THREAD

**Discussion by hypervtechnics** [renovatebot/renovate#17194] [Request Help]
There are cases like with the azure pipelines manager where the extracted dependencies somehow rely on the repository which is currently inspected. I'd like to submit a PR to fix a well wanted feature (extracting git repository resources from a pipeline yaml) and need that functionality for that. Issue is #15028

I can either calculate it by using platform + endpoint or infer it from the current clone url from the repository details.

**Comment by rarkins** [comment]
We prefer to not pass those to the extract config. It would be better if the relative link can be resolved in datasource

**Reply by hypervtechnics** [reply]
How would scenarios like the given one be handled in the architecture then? Because we can not infer the repository path only from the package file itself. I mean it's a configuration to configure renovate so I'd say: Make it accessible everywhere 😄
